packages/core/src/modules/event/event-service.ts

```diff
@@ -20,19 +21,25 @@ export interface EventService {
 export class EventServiceImpl implements EventService {
   constructor(
     private readonly eventRepository: EventRepository,
-    private readonly attendanceRepository: AttendanceRepository
+    private readonly attendanceRepository: AttendanceRepository,
+    private readonly committeeOrganizerRepository: EventCommitteeRepository
   ) {}
 
-  async createEvent(eventCreate: EventWrite): Promise<Event> {
+  async createEvent(eventCreate: EventWrite, committees: string[]): Promise<Event> {
     const event = await this.eventRepository.create(eventCreate)
+
```

Comment:
A lot of unnecessary whitespace added here maybe you forgot to swap eslint configs